sop/modules/OSD/pages/CephOSDDiskNotResponding.adoc

```diff
@@ -1,47 +1,45 @@
-// begin header
-ifdef::env-github[]
-:tip-caption: :bulb:
-:note-caption: :information_source:
-:important-caption: :heavy_exclamation_mark:
-:caution-caption: :fire:
-:warning-caption: :warning:
-endif::[]
-:numbered:
-:toc: macro
-:toc-title: pass:[<b>Table of Contents</b>]
-// end header
-= CephOSDDiskNotResponding
+include::partial$alert_header.adoc[]
 
-toc::[]
+:alertname: CephOSDDiskNotResponding
+:message: Disk not responding
+:description: Disk device {{ $labels.device }} not responding on host {{ $labels.host }}.
+:detailed_description: Detailed Description: A disk device is not responding. Check alert details for disk and host. Replace disk if necessary.
 
-== Check Description 
+:resolution_description:
+:severity: Error
+:ceph-component: rook-ceph-mgr
 
-*Severity:* Error
+= {alertname}
 
-*Potential Customer Impact:* High
+toc::[]
 
+== Check Description
+*Severity:* {severity}
+
+*Potential Customer Impact:* High
 
 == Overview
+{description}
 
-A disk device is not responding. Check alert details for disk and host. Replace disk if necessary.
+{detailed_description}
 
 == Prerequsities
 
 include::partial$prerequisites.adoc[]
 
 ==  Alert
-include::partial$gather_logs.adoc[]
-
-
 
+=== Make changes to solve alert
 
+The missing/unresponsive disk(volume) must be replaced as soon as possible. This procedure determines which node is having disk issues.
 
+==== Determine OCS Nodes
+WIP
```

Comment:
Do you need a way to figure out which node the OSD pod is on?

Reply:
I started to write this SOP when I thought they might already have a disk not responding SOP.  I'll try to find out.